Library/include/CSP/Multiplayer/SpaceEntity.h

```diff
@@ -206,6 +207,13 @@ class CSP_API SpaceEntity
 	/// @return SpaceEntitySystem
 	SpaceEntitySystem* GetSpaceEntitySystem();
 
+	void SetParentEntity(SpaceEntity* Parent);
+	void RemoveParentEntity();
```

Comment:
If you feel that this would result in a cleaner API, could we remove this if we were to allow `MyEntity->SetParentEntity(nullptr)`?

Reply:
I like it

Reply:
Latest changes look good, thanks!